tests/scripts/analyze_outcomes.py

```diff
@@ -197,13 +199,15 @@ def run(self, results: Results, outcomes: Outcomes) -> None:
             hit = any(suite_case in comp_outcomes.successes or
                       suite_case in comp_outcomes.failures
                       for comp_outcomes in outcomes.values())
+            (test_suite, test_description) = suite_case.split(';')
+            ignored = self.is_test_case_ignored(test_suite, test_description)
 
-            if not hit and suite_case not in self.ALLOW_LIST:
+            if not hit and not ignored:
                 if self.full_coverage:
                     results.error('Test case not executed: {}', suite_case)
                 else:
                     results.warning('Test case not executed: {}', suite_case)
-            elif hit and suite_case in self.ALLOW_LIST:
+            elif hit and ignored:
                 # Test Case should be removed from the allow list.
                 if self.full_coverage:
                     results.error('Allow listed test case was executed: {}', suite_case)
```

Comment:
Nit: this is no longer completely correct - an _ignored_ test was executed, not an Allowed test.